packages/k-ramel/typing/index.d.ts

```diff
@@ -1,54 +1,65 @@
-declare module 'k-ramel' {
-  import { TypeDefinition, TypeDefinitionFunction } from 'k-redux-factory'
-  import { compose, applyMiddleware, Dispatch, enhanceMiddleware } from 'redux'
-  export { Types } from 'k-redux-factory'
+declare module "k-ramel" {
+  import {
+    TypeDefinition,
+    TypeDefinitionFunction,
+    ReducerType
+  } from "k-redux-factory";
+  import { compose, applyMiddleware, Dispatch, enhanceMiddleware } from "redux";
+  export { Types } from "k-redux-factory";
   export type StoreOptions = {
-    hideRedux?: boolean
-    enhancer?: any
-    init?: any
-    listeners?: any
-    devtools?: any
-    trace?: boolean
-    traceLimit?: number
-    name?: string
-    drivers?: any
-  }
+    hideRedux?: boolean;
```

Comment:
are you sure about all the ";" here?

Reply:
eslint did this, I'm not quite sure if this is wrong or right, well according to the documentation it seems fine.

Reply:
Can you run the build, then I merge, and you can test the master version?

Reply:
done ;)